ignite/services/scaffolder/scaffolder.go

```diff
@@ -64,7 +64,8 @@ func New(appPath string) (Scaffolder, error) {
 }
 
 func finish(ctx context.Context, cacheStorage cache.Storage, path, gomodPath string) error {
-	if err := protoc(ctx, cacheStorage, path, gomodPath); err != nil {
+	err := protoc(ctx, cacheStorage, path, gomodPath)
```

Comment:
Why remove these inline error if above?
IMHO, I think it is more clear, but it's up to you; WDYT?

Reply:
@jeronimoalbi ?

Reply:
Not sure why it was changed by me but I agree with @Pantani that it looks better inline.

Personally I usually remove inlining when there are too many arguments so the line gets too long or harder to read, or otherwise because of consistency with other cases within the scope, which is not the case here.